tests/lib/DefaultOsTest.php

```diff
@@ -207,4 +208,42 @@ public function testGetDiskInfoInvalidCommandOutput(): void {
 	public function testSupported(): void {
 		$this->assertTrue($this->os->supported());
 	}
+
+	public function testGetNetworkInterfaces(): void {
+		$this->os->method('getNetInterfaces')
+			->willReturn(include __DIR__ . '/../data/linux_net_get_interfaces.php');
```

Comment:
This weird, there is no better way? Maybe json_decode a json file instead?

Reply:
Good question.

`getNetInterfaces` is a wrapper for `net_get_interfaces`  to mock the network interfaces for testing. linux_net_get_interfaces.php is the result of var_export(net_get_interfaces());

Right now, it's as close as it gets to the original return. 
It should work to export as json and decode for the test again.  

What do you prefer?

Reply:
I would feel better with json, yes. This requires to go read the PHP file to check what it does. A json_decode call will be clearer.

Reply:
Okay.

Reply:
Done.